src/libs/Timers.ts

```diff
@@ -1,23 +1,18 @@
-import _ from 'underscore';
-
-const timers = [];
+const timers: number[] = [];
 
 /**
  * Register a timer so it can be cleaned up later.
- *
- * @param {Number} timerID
- * @returns {Number}
  */
-function register(timerID) {
+function register(timerID: number): number {
```

Comment:
Shouldn't timerID be of type NodeJS.Timer also same for return type?

Reply:
@BartoszGrajdek thoughts on this before we merge?